client/src/components/Breadcrumbs.jsx

```diff
@@ -1,25 +1,28 @@
 import './Breadcrumbs.css'
 
+const pathToBreadcrumbs = (currentPath) => {
+  const paths = currentPath.split('/').slice(1)
+  const breadcrumbs = [{ name: 'home', path: '/' }]
+  let currentFullPath = ''
+
+  paths.forEach((name) => {
+    currentFullPath += `/${name}`
+    breadcrumbs.push({ name, path: currentFullPath })
+  })

```

Comment:
so here is your logic: 
1. get an array of "something" out of "currentPath"
2. make an array of "something else" with first element in it already set
3. declare some variable
4. loop through the array of "something" and update another array of "something else" and for each use that some variable.

refactor this logic into this:
1. check if "currentPath" is "/" if yes, return an array with only one object: `[{name: "home", path: "/"}]`
2. if not, then apply transformation on "currentPath" by splitting it, slicing, and mapping each element to an object of `{name, path}`. then return the result.